fvm/transactionInvoker.go

```diff
@@ -205,22 +205,14 @@ func (i *TransactionInvoker) Process(
 			// drop delta
 			childState.View().DropDelta()
 			programs.Cleanup(nil)
-			i.logger.Info().
+			i.logger.Warn().
```

Comment:
This is an expected error in the case where the funds of the payer approach 0. I am not sure a warn level is appropriate here.

Reply:
Both the error message and comments sound like the error was unexpected.

If the error is expected, is there a specific error type that we can compare and print a info log saying something like "payer has insufficient transaction fee"

I will revert this change for now.

Reply:
Created a bug report https://github.com/dapperlabs/flow-go/issues/6123